src/account-db.js

```diff
@@ -49,8 +49,13 @@ export function bootstrap(password) {
 }
 
 export function login(password) {
+  if (password == null || password === '') {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind following strict JS here?
```suggestion
  if (password === null || password === '') {
```

Reply:
I think that it actually comes through as `undefined` so the weak comparison to `null` was required here to pick out null-ish. I can change it to be `password === undefined` if you'd prefer but I was trying to keep it consistent with similar checks in the same file.

Reply:
We could do both `password === undefined || password === null`. Eventually we will refactor the entire repo to be strict.. but for now we can focus only on the new code :)